### PR TITLE
[quest] The Ulduar quests map markers should be now visible.

### PR DESCRIPTION
**Pull request by Cabro**
Trying a way to make Questie show map markers inside the raid maps. It involves using some fake IDs. I would like for someone else to double check and test this PR. It all works great locally, and I tested with modifying some loot tables to also include the objects and NPCs in these changes, and they were drawn correctly.

Also, if you log in outside of Ulduar, the quest map markers should also be visible in the correct map levels of Ulduar when you enter.

If this fix seems to be good, then I can add map markers for the rest of the dungeons.

### CODE REVIEW

File: Database/Corrections/wotlkNPCFixes.lua [hunk 1]
```diff
@@ -984,6 +984,10 @@ function QuestieWotlkNpcFixes:Load()
             [npcKeys.questStarts] = {13500},
             [npcKeys.questEnds] = {13500},
         },
+        [33288] = {
+            [npcKeys.spawns] = {[zoneIDs.ULDUAR]={{-1,-1}},[zoneIDs.THE_PRISON_OF_YOGG_SARON]={{68,40.2}}},
```
Comment (BreakBB) [inline Database/Corrections/wotlkNPCFixes.lua:988]: I am not sure if these spawns should be the other way around in order to work 🤔 I have something in mind that a loop stops when encountering a `-1,-1` spawn.

Comment (Cabro) [reply Database/Corrections/wotlkNPCFixes.lua:988]: They were showing just fine when i was tweaking them, but i'll change the order

**Comment by BreakBB** [comment]
Can this be merged @Cabro ?

**Comment by Cabro** [comment]
They all work locally for me, no issues. Just wanted someone to double check the `requiredSpell` changes in the function for logic, and whether someone had a problem with the Ulduar quests, like nil errors or something.

**Comment by Muehe** [comment]
Merged by manual rebase, see fe6c56a00019153f62a841c393089d93d8e1af54 and its 6 parent commits.

**Pull request closed without merging**